src/fhirtypes/CodeSystem.ts

```diff
@@ -52,7 +53,7 @@ export class CodeSystem {
    * @returns {string} the filename
    */
   getFileName(): string {
-    return `CodeSystem-${this.id}.json`;
+    return sanitize(`CodeSystem-${this.id.replace(/\/\\/g, '-')}.json`, { replacement: '-' });
```

Comment:
Do we still need the `replace` with the regular expression if it is inside `sanitize`?

Reply:
Doh.  I had started by using a replace and then switched to sanitize and forgot to remove that!